Stripe/PublicHeaders/STPCustomer.h

```diff
@@ -50,6 +52,11 @@ NS_ASSUME_NONNULL_BEGIN
  */
 @property(nonatomic, readonly) NSArray<id<STPSourceProtocol>> *sources;
 
+/**
+ *  The customer's shipping address.
+ */
+@property(nonatomic, readonly) STPAddress *shipping;
```

Comment:
just `shipping` is a bit confusing. I think we call this `shippingAddress` in other places.